app/views/content/ways-to-train.md

```diff
@@ -148,13 +148,17 @@ Fees are usually around £9,250 but you can:
 
 ### Earn a salary
 
-Some teacher training programmes like 'school direct (salaried)' and 'postgraduate teaching apprenticeships' offer a salary.
+Some teacher training programmes offer you a salary as you train. These include 'school direct (salaried)' and 'postgraduate teaching apprenticeships'.
 
-These courses are usually suitable for people who already have relevant teaching experience, or are already working as unqualified teachers. 
+These salaried routes are usually suitable for people with a degree and relevant school experience. 
 
 Many 'school direct (salaried)' programmes give you a PGCE or PGDE (an academic qualification) in addition to QTS.
 
-Courses typically take one year but you may be able to study part-time.
+It can be easy to confuse the 'school direct (salaried)' programme with the non-salaried 'school direct' programme. Make sure you check with your chosen provider before applying.
+
+Courses usually take one year but you may be able to study part-time.
```

Comment:
This sentence feels a little odd to me, the second part doesn't feel related to the first.

Perhaps:

```suggestion
Courses usually take one year but you may take two if you study part-time.
```
Just an example, don't know if it's even true! 😅

Reply:
Not sure I agree with your suggestion @peteryates but will leave for @Tomcowan1 as I know that he, Ellie and Tara are looking at broader changes to the Ways to train page as a result of TTA feedback

Reply:
Maybe: "Full-time courses usually take one year to complete but you may be able to study part-time." 
Or: "Courses are usually full-time and take one year to complete. You may be able to study part-time."